packages/gamut-labs/src/AppHeader/AppHeaderElements/AppHeaderLogo/index.tsx

```diff
@@ -24,7 +53,7 @@ export const AppHeaderLogo: React.FC<AppHeaderLogoProps> = ({
     >
       <Logo
         color="currentColor"
-        variant={item.pro ? 'pro' : 'default'}
+        variant={getVariant()}
```

Comment:
Instead of this helper func, could we get the same thing in a single ternary?
```suggestion
        variant={item.checkMini && showMini ? 'mini' : item.pro ? 'pro' : 'default'}
```